### PR TITLE
pack readme - [dev - bug]

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45655

## Description
This PR is opened for a dev bug.
Inorder to validate empty pack readme files only on packs with playbooks. (not for all use cases like now).
Also add a more informative error message.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py [hunk 1]
```diff
@@ -230,12 +230,13 @@ def validate_author_image_exists(self):
...
-        Validates that README.md file is not empty for partner packs and packs with use cases
+        Validates that README.md file is not empty for partner packs and packs with playbooks
         """
-        if (self.support == 'partner' or self._contains_use_case()) and self._check_if_file_is_empty(self.readme_file):
+        playbooks_path = os.path.join(self.pack_path, "Playbooks")
+        contains_playbooks = os.path.exists(playbooks_path) and len(os.listdir(playbooks_path)) != 0
+        if (self.support == 'partner' or contains_playbooks) and self._check_if_file_is_empty(self.readme_file):
             if self._add_error(Errors.empty_readme_error(), self.readme_file):
                 return False
-
```
Comment (yaakovi) [inline demisto_sdk/commands/common/hook_validations/pack_unique_files.py:238]: restore the new line

File: demisto_sdk/commands/common/tests/pack_unique_files_test.py [hunk 2]
```diff
@@ -562,6 +564,24 @@ def test_validate_pack_readme_file_is_not_empty_use_case(self, mocker, text, res
...
+    def test_no_readme_alert_on_scripts_layouts(self, repo, mocker):
+        """
+          Given:
+               - pack with scripts or layouts
+
+           When:
+               - Running test_validate_pack_readme_file_is_not_empty_partner.
+
+           Then:
+               - Ensure no validation on empty pack read me file.
```
Comment (yaakovi) [inline demisto_sdk/commands/common/tests/pack_unique_files_test.py:576]:
```suggestion
               - Ensure no error on an empty pack README file.
```

**Review by yaakovi — APPROVED** [review]

**Comment by idovandijk** [comment]
@omerKarkKatz can you please check whether this is also failing on existing packs, and if so which ones (I just want to validate that the behavior makes sense)

**Comment by omerKarkKatz** [comment]
@idovandijk All the existing pack's that don't apply to this rule, include an ignore error code of RM104 this is the error code of this issue. The main point of the error is to effect future integrations.
if this error was not in allowed ignored errors the following packs will fail:

Packs/Accessdata/README.md - [RM104]
Packs/Akamai_WAF/README.md - [RM104]
Packs/AlphaSOC_Network_Behavior_Analytics/README.md - [RM104]
Packs/AlphaSOC_Wisdom/README.md - [RM104]
Packs/AlphaVantage/README.md - [RM104]
Packs/Anomali_Enterprise/README.md - [RM104]
Packs/Anomali_ThreatStream/README.md - [RM104]
Packs/Ansible_Powered_Integrations/README.md - [RM104]
Packs/AnsibleTower/README.md - [RM104]
Packs/ANYRUN/README.md - [RM104]
Packs/ArcherRSA/README.md - [RM104]
Packs/ArcSightESM/README.md - [RM104]
Packs/ARIAPacketIntelligence/README.md - [RM104]
Packs/AWS-IAM/README.md - [RM104]
Packs/Axonius/README.md - [RM104]
Packs/AzureLogAnalytics/README.md - [RM104]
Packs/BitDam/README.md - [RM104]
Packs/BPA/README.md - [RM104]
Packs/C2sec/README.md - [RM104]
Packs/Carbon_Black_Enterprise_Live_Response/README.md - [RM104]
Packs/CarbonBlackEnterpriseEDR/README.md - [RM104]
Packs/CarbonBlackProtect/README.md - [RM104]
Packs/CheckpointFirewall/README.md - [RM104]
Packs/CiscoWebExFeed/README.md - [RM104]
Packs/CloudConvert/README.md - [RM104]
Packs/CloudShark/README.md - [RM104]
Packs/Cognni/README.md - [RM104]
Packs/CommonPlaybooks/README.md - [RM104]
Packs/Coralogix/README.md - [RM104]
Packs/CounterCraft/README.md - [RM104]
Packs/CrowdStrikeFalcon/README.md - [RM104]
Packs/CrowdStrikeFalconSandbox/README.md - [RM104]
Packs/CrowdStrikeFalconX/README.md - [RM104]
Packs/CrowdStrikeHost/README.md - [RM104]
Packs/CrowdStrikeMalquery/README.md - [RM104]
Packs/CTIX/README.md - [RM104]
Packs/CuckooSandbox/README.md - [RM104]
Packs/CVESearch/README.md - [RM104]
Packs/Cybereason/README.md - [RM104]
Packs/CyberTotal/README.md - [RM104]
Packs/Cylance_Protect/README.md - [RM104]
Packs/D2/README.md - [RM104]
Packs/DeepInstinct/README.md - [RM104]
Packs/DeprecatedContent/README.md - [RM104]
Packs/Digital_Defense_FrontlineVM/README.md - [RM104]
Packs/DigitalGuardian/README.md - [RM104]
Packs/EDL/README.md - [RM104]
Packs/epo/README.md - [RM104]
Packs/Exchange2016_Compliance/README.md - [RM104]
Packs/Expanse/README.md - [RM104]
Packs/FeedAWS/README.md - [RM104]
Packs/FeedAzure/README.md - [RM104]
Packs/FeedCyjax/README.md - [RM104]
Packs/FeedOffice365/README.md - [RM104]
Packs/fireeye/README.md - [RM104]
Packs/FireEyeHelix/README.md - [RM104]
Packs/FireEyeHX/README.md - [RM104]
Packs/FortiManager/README.md - [RM104]
Packs/FortiSandbox/README.md - [RM104]
Packs/Gmail/README.md - [RM104]
Packs/GoogleCloudCompute/README.md - [RM104]
Packs/GoogleCloudSCC/README.md - [RM104]
Packs/GoogleKubernetesEngine/README.md - [RM104]
Packs/GoogleVault/README.md - [RM104]
Packs/HelloWorld/README.md - [RM104]
Packs/Hunting/README.md - [RM104]
Packs/HybridAnalysis/README.md - [RM104]
Packs/ILert/README.md - [RM104]
Packs/illuminate/README.md - [RM104]
Packs/Infocyte/README.md - [RM104]
Packs/IronDefense/README.md - [RM104]
Packs/JoeSecurity/README.md - [RM104]
Packs/Kenna/README.md - [RM104]
Packs/LSASSCredentialDumping/README.md - [RM104]
Packs/McAfee-MAR/README.md - [RM104]
Packs/McAfee-TIE/README.md - [RM104]
Packs/McAfee_Advanced_Threat_Defense/README.md - [RM104]
Packs/McAfee_DXL/README.md - [RM104]
Packs/MicrosoftDefenderAdvancedThreatProtection/README.md - [RM104]
Packs/Mimecast/README.md - [RM104]
Packs/NCSCCyberAsssessmentFramework/README.md - [RM104]
Packs/NutanixHypervisor/README.md - [RM104]
Packs/Okta/README.md - [RM104]
Packs/OnboardingIntegration/README.md - [RM104]
Packs/PaloAltoNetworks_PAN_OS_EDL_Management/README.md - [RM104]
Packs/PaloAltoNetworks_Threat_Vault/README.md - [RM104]
Packs/PaloAltoNetworksAutomaticSLR_Community/README.md - [RM104]
Packs/PANOStoCDLMonitoring/README.md - [RM104]
Packs/PerceptionPoint/README.md - [RM104]
Packs/PhishAI/README.md - [RM104]
Packs/PhishLabs/README.md - [RM104]
Packs/PolySwarm/README.md - [RM104]
Packs/PowershellPayloadResponse/README.md - [RM104]
Packs/ProofpointServerProtection/README.md - [RM104]
Packs/ProofpointThreatResponse/README.md - [RM104]
Packs/Rundeck/README.md - [RM104]
Packs/SecBI/README.md - [RM104]
Packs/SecurityIntelligenceServicesFeed/README.md - [RM104]
Packs/ServiceDeskPlus/README.md - [RM104]
Packs/ShiftManagement-AssignToNextShift/README.md - [RM104]
Packs/Skyformation/README.md - [RM104]
Packs/Slack/README.md - [RM104]
Packs/SNDBOX/README.md - [RM104]
Packs/SophosXGFirewall/README.md - [RM104]
Packs/SymantecBlueCoatMalwareAnalysis/README.md - [RM104]
Packs/Tanium/README.md - [RM104]
Packs/TaniumThreatResponse/README.md - [RM104]
Packs/TeamManagement/README.md - [RM104]
Packs/Tenable_io/README.md - [RM104]
Packs/Tenable_sc/README.md - [RM104]
Packs/Tidy/README.md - [RM104]
Packs/TIMCampaignTracking/README.md - [RM104]
Packs/Traps/README.md - [RM104]
Packs/Trello/README.md - [RM104]
Packs/TrendMicroApex/README.md - [RM104]
Packs/TrendMicroCAS/README.md - [RM104]
Packs/TrendMicroDDA/README.md - [RM104]
Packs/trendMicroDsm/README.md - [RM104]
Packs/Tripwire/README.md - [RM104]
Packs/TruSTAR/README.md - [RM104]
Packs/Tufin/README.md - [RM104]
Packs/UnifiVideoNVR/README.md - [RM104]
Packs/Uptycs/README.md - [RM104]
Packs/Zimperium/README.md - [RM104]
Packs/Zscaler/README.md - [RM104]

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py (hunk 1 above)
Comment (omerKarkKatz) [reply demisto_sdk/commands/common/hook_validations/pack_unique_files.py:238]: Done

**Comment by dorschw** [comment]
there's no CHANGELOG.md here, was this on purpose? @yaakovi @omerKarkKatz

**Comment by omerKarkKatz** [comment]
@dorschw i don't think that it was intentional i just forgot (not in my system yet). 
i will open a PR  and add one. Sorry, wont happen again! ;)

**Pull request merged**